artemis-server/src/main/java/org/apache/activemq/artemis/core/config/WildcardConfiguration.java

```diff
@@ -240,4 +240,22 @@ private void replaceChar(StringBuilder result, char replace, char replacement) {
          }
       }
    }
+
+   /**
+    * Checks if the given input string represents a valid character that matches one of the defined wildcard characters.
+    * <p>
+    * The input string must contain exactly one character. If the input is null, empty, or contains more than one
+    * character, an IllegalArgumentException is thrown.
+    *
+    * @param input the input string to check; it must contain exactly one character
+    * @return true if the single character in the input matches one of the wildcard characters, otherwise false
+    * @throws IllegalArgumentException if the input is null, empty, or contains more than one character
+    */
+   public boolean contains(String input) {
+      if (input == null || input.isEmpty() || input.length() > 1) {
+         throw new IllegalArgumentException();
+      }
+      char inputChar = input.charAt(0);
```

Comment:
Since the value passed in the only usage of this method must be a fixed String with single character, so should the param here perhaps just be made a `char` (prob need another static constant, but they can inter-reference) and then none of these checks or the conversion or related documentation are needed?

Reply:
I considered that originally but decided against it for some reason(s) that I can no longer recall. I'm using `char` now, and it's much nicer.